app/controllers/survey_controller.rb

```diff
@@ -1,5 +1,40 @@
 class SurveyController < ApplicationController
   def index
+    @survey = Survey.new
+  end
+
+  def create
+    current_user.feelings.create(params['feeling'])
+
+    params['time'].each do |time|
+      current_user.time_preference.create("#{time}": true)
+    end
+
+    params['music'].each do |music|
+      if music && current_user.music_prefence[music]
+        current_user.music_preference.udpate("#{music}": true)
+      elsif music && !current_user.music_preference['custom']
+        current_user.music_prefence.update(custom: music)
+      elsif music && current_user.music_preference['custom']
+        current_user.music_prefence.update(custom2: music)
+      end
+    end
+
+    params['resources'].each do |resource|
+      current_user.activity_preferences.create(description: resource)
```

Comment:
To remain consistent, we should either make each type of `_preference` method singular or plural.

Reply:
I was thinking there would be a difference when you are creating but I will change it to singular @freeheeling